src/features/previewManager.ts

```diff
@@ -45,10 +45,10 @@ export class AsciidocPreviewManager implements vscode.WebviewPanelSerializer {
     }
   }
 
-  public preview (
+  public async preview (
     resource: vscode.Uri,
     previewSettings: PreviewSettings
-  ): void {
+  ): Promise<void> {
```

Comment:
I don't think this method needs to be `async` since we are not using `await` keyword in the method body.